Util/AndroidManifestMerger/build.cake

```diff
@@ -62,7 +62,8 @@ Task("tests")
 	//       as ensuring that the target is properly skipped on subsequent builds.
 
 	var android = (XNamespace)"http://schemas.android.com/apk/res/android";
-	var xdoc = XDocument.Load("./samples/FancyMergingApp/obj/Release/90/android/AndroidManifest.xml");
+	var files = GetFiles("./samples/FancyMergingApp/obj/Release/**/android/AndroidManifest.xml");
+	var xdoc = XDocument.Load(files.Single().FullPath);
```

Comment:
It appears that macOS uses the path `obj/Release/90/xxx` and Windows uses the path `obj/Releaase/xxx`